core/src/main/java/smashdudes/ecs/systems/CharacterJumpInputSystem.java

```diff
@@ -30,13 +26,32 @@ protected void updateEntity(Entity entity, float dt)
         CharacterInputComponent i = entity.getComponent(CharacterInputComponent.class);
         VelocityComponent v = entity.getComponent(VelocityComponent.class);
         JumpComponent j = entity.getComponent(JumpComponent.class);
+        InAirComponent a = entity.getComponent(InAirComponent.class);
+        OnGroundComponent g = entity.getComponent(OnGroundComponent.class);
```

Comment:
This system is not registered with these components. 
Since can jump in air and while on ground this does not seem appropriate

Reply:
yeah, not too sure how I wanna deal with that but I agree